### ISSUE TITLE
Incompatible with newly released Werkzeug 1.0.0

### ISSUE THREAD

**Issue by jsnod** [puckel/docker-airflow#499]
v1.0.0 of Werkzeug was just released, and it now breaks builds with:

`ImportError: cannot import name 'secure_filename' from 'werkzeug'`

According to the [changelog](https://werkzeug.palletsprojects.com/en/1.0.x/changes/), top-level attributes were removed in 1.0:

```
Deprecate most top-level attributes provided by the werkzeug module in favor of 
direct imports. The deprecated imports will be removed in version 1.0. For example, 
instead of import werkzeug; werkzeug.url_quote, do from werkzeug.urls 
import url_quote. 
```

The workaround for now is to pin the old version in the `Dockerfile`:

`pip install werkzeug==0.16.1`

**Comment by liorhar** [comment]
Airflow version 1.10.9 fixes that. Maybe this project needs to upgrade to resolve this issue.
https://airflow.apache.org/docs/stable/changelog.html#airflow-1-10-9-2020-02-10

**Comment by puckel** [comment]
@jsnod It's already "fixed" in docker-ariflow 1.10.8 cf 0d9b032

**Comment by rahulj51** [comment]
Should this fix not be back ported to previous versions? I am trying to deploy an older version (1.10.4) and still getting tis error.
